www/jekyll/_layouts/default.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+<meta http-equiv="Content-Type"
+      content="text/html; charset=UTF-8" />
+<title>{{ site.title }}</title>
+{% unless page.plain %}
+<link rel="stylesheet"
+      type="text/css"
+      href="/assets/css/style.css" />
+{% endunless %}
+<link rel="stylesheet"
+      type="text/css"
+      href="/spec.css" />
+</head>
+
+<body>
+<main id="content" class="main-content" role="main">
+<div class="wrapper">
+<section>
+
+{{content}}
+
+</section>
+</div>
+</main>
+
+<script>
+var i, s, l;
+s = document.querySelectorAll('code > code');
+for (i = 0, l = s.length; i < l; i++) {
+  s[i].parentElement.classList.add('container');
+}
```

Comment:
```suggestion
for (const c of document.querySelectorAll('code > code')) {
  c.parentElement.classList.add('container');
}
```

This is simpler, and doesn't pollute the global namespace.

Reply:
That's nice. My JS fu isn't up to date.